package.json

```diff
@@ -77,15 +77,15 @@
   "size-limit": [
     {
       "path": "./packages/runtime/dist/browser/index.js",
-      "limit": "105B",
+      "limit": "114B",
```

Comment:
hmm why did this go up?

Reply:
`es6` hehe. I am moving it back to `es5` And removing all spread stuff. We will target `es5` only.